Description: Use renamed factory library from Debian singular package.
 See [1].  We also add -I/usr/include/singular to CPPFLAGS in debian/rules.
 .
 [1] http://anonscm.debian.org/cgit/debian-science/packages/singular.git/tree/
     debian/patches/debianization-rename-libraries.patch
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-06-03

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -1268,8 +1268,8 @@
 BUILD_gftables=yes		dnl it seems that factory now includes gftables, and installs it
 
 if test "$DEBUG" = yes
-then SINGULARLIBS="-lfactory  "
-else SINGULARLIBS="-lfactory  "
+then SINGULARLIBS="-lsingular-factory  "
+else SINGULARLIBS="-lsingular-factory  "
 fi
 while test $BUILD_factory = no 
 do AC_LANG(C++)
